examples/apply-setters-simple/.expected/diff.patch

```diff
@@ -1,25 +1,29 @@
 diff --git a/resources.yaml b/resources.yaml
-index f9d3714..76fef40 100644
+index b521905..0d10662 100644
 --- a/resources.yaml
 +++ b/resources.yaml
-@@ -1,15 +1,15 @@
- apiVersion: v1
- kind: ConfigMap
+@@ -3,7 +3,7 @@ kind: Deployment
  metadata:
--  name: ubuntu-development # kpt-set: ${image}-development
-+  name: darwin-development # kpt-set: ${image}-development
- data:
-   some-key: some-value
- ---
- apiVersion: v1
- kind: MyKind
+   name: my-nginx
+ spec:
+-  replicas: 4 # kpt-set: ${nginx-replicas}
++  replicas: 3 # kpt-set: ${nginx-replicas}
```

Comment:
why is doc changes changing values?

Reply:
I have updated the examples to be more relevant. Used nginx deployment resource.